### PR TITLE
Add some more lines to the UG

### PR DESCRIPTION
**Pull request by junghon3709**
Also mentioned what "last seen list" is since we throw around that term a lot

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -797,6 +797,7 @@ Extensions:
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
 * **Private contact detail**: A contact detail that is not meant to be shared with others
 * **Involvement**: A label attached to each student/teacher specifying the person's involvement with the User.
+* **Currently displayed list**: for students, for example, there are two lists that are maintained by the application. One is the full list of students. The other is the list which the user currently sees on screen. For various reasons, the list the user sees may not be the same as the full list of students (perhaps the user executed a `findStudent` command) . When we say "currently displayed list" we mean the list that is currently shown to the user.
```
Comment (nniiggeell) [inline docs/DeveloperGuide.md:800]: should we caps the first letter "For students..."

File: docs/UserGuide.md [hunk 2]
```diff
@@ -140,6 +140,8 @@ Action | Format | Window
   class as `in my math class`. A CCA teacher might store the involvement of a student in his badminton CCA as `badminton`.
   A teacher involved in an overseas CIP trip might want to store the involvement of students going along with him as `CIP trip`.
 
+* **Currently displayed list**: for students, for example, there are two lists that are maintained by the application. One is the full list of students. The other is the list which the user currently sees on screen. For various reasons, the list the user sees may not be the same as the full list of students (perhaps the user executed a `findStudent` command) . When we say "currently displayed list" we mean the list that is currently shown to the user.
```
Comment (nniiggeell) [inline docs/UserGuide.md:143]: same here, might want to caps the first letter

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (junghon3709) [reply docs/DeveloperGuide.md:800]: Hmm yeah good point, I'll fix it

**Comment by junghon3709** [comment]
Alright, it has been fixed.

**Review by g4ryy — APPROVED** [review]
I shall merge this since all issues have been fixed

**Pull request merged**